subprojects/gradle/build-metrics/src/main/kotlin/com/avito/android/plugin/build_metrics/internal/BuildOperationsResult.kt

```diff
@@ -0,0 +1,20 @@
+package com.avito.android.plugin.build_metrics.internal
+
+internal class BuildOperationsResult(
+    val tasksExecutions: List<TaskExecutionResult>,
+    val cacheOperations: CacheOperations
+)
+
+internal class CacheOperations(
```

Comment:
```suggestion
internal class CacheOperation(
```
?

Reply:
Not sure, there will be statistics for all operations in a build related to cache.